backend/src/main/kotlin/fr/gouv/cacem/monitorenv/domain/use_cases/missions/DeleteMission.kt

```diff
@@ -1,17 +1,19 @@
-@file:Suppress("ktlint:standard:package-name")
```

Comment:
je sais pas ce que c'est ça

Reply:
c'est pour le linter back. Le nom du package ne devrait pas être `use_case` mais `useCase`. On retrouve donc cette ligne dans tous les fichiers sous `domain/use_case`

Reply:
@thoomasbro, il y a plein de useCase qui n'ont pas cette ligne (dossier amps, controlUnit, Administration, ControlPlan,...)

Reply:
Effectivement... j'étais persuadé d'avoir retiré `-e "Package name must not contain underscore"`du linter. J'avais ajouté ces annotations dans ce but. Je serais pour remettre cette annotation.